src/app/dashboard/events/[id]/page.tsx

```diff
@@ -1,8 +1,92 @@
+import { format, getHours, getMinutes } from "date-fns";
+import {
+  Calendar1,
+  CalendarX,
+  Share2Icon,
+  SquarePenIcon,
+  User,
+  Users,
+} from "lucide-react";
+import Image from "next/image";
+import Link from "next/link";
+import { notFound, unauthorized } from "next/navigation";
+
+import EventPhotoPlaceholder from "@/../public/event-photo-placeholder.png";
+import { Button } from "@/components/ui/button";
+import { API_URL, PHOTO_URL } from "@/lib/api";
+import { verifySession } from "@/lib/session";
+import type { Event } from "@/types/event";
+
 export default async function DashboardEventPage({
   params,
 }: {
   params: Promise<{ id: string }>;
 }) {
-  const awaited = await params;
-  return <h1 className="text-3xl font-bold">Event o ID {awaited.id}</h1>;
+  const session = await verifySession();
+  if (session == null || typeof session.bearerToken !== "string") {
+    unauthorized();
+  }
+  const { bearerToken } = session;
+  const { id } = await params;
+  const response = await fetch(`${API_URL}/events/${id}`, {
+    headers: {
+      Authorization: `Bearer ${bearerToken}`,
+    },
+  });
+  if (!response.ok) {
+    notFound();
+  }
+  const event = (await response.json()) as Event;
```

Comment:
```suggestion
  const { bearerToken } = session;
  const { id } = await params;
  
  const response = await fetch(`${API_URL}/events/${id}`, {
    headers: {
      Authorization: `Bearer ${bearerToken}`,
    },
  });
  
  if (!response.ok) {
    notFound();
  }
  
  const event = (await response.json()) as Event;
```
Spacing poprawia czytelność kodu